### ISSUE TITLE
css not minified

### ISSUE THREAD

**Issue by AccessViolator** [ducksoupdev/vue-webpack-typescript#36]
Thanks for the template, it's the only one among the ones I saw, not having babel and eslint, looks like the most proper typescript setup so far :)

I've just scaffolded a new project, ran `npm run build` and I see that dist/css/main.#.css is not minified (though "treeshaken"). I had this warning printed a few times during the build process, not sure if it's important, I'm new to webpack:
```
    WARNING in ./node_modules/css-loader?{"minimize":true,"sourceMap":true,"importLoaders":2}!./node_modules/postcss-loader/lib?{}!./node_modules/sass-loader/lib/loader.js?{"outputStyle":"expanded","sourceMap":true,"sourceMapContents":true}!./src/sass/main.scss
    (Emitted value instead of an instance of Error)

     ⚠️  PostCSS Loader

    Previous source map found, but options.sourceMap isn't set.
    In this case the loader will discard the source map entirely for performance reasons.
    See https://github.com/postcss/postcss-loader#sourcemap for more information.
```

On a side note - I think it's worth updating bootstrap to v4.

**Comment by AccessViolator** [comment]
To get rid of the warning I added `sourceMap: true` to postcss-loader configuration
```diff
        {
          loader: 'postcss-loader',
          options: {
            plugins: () => [autoprefixer],
+           sourceMap: true
          }
        },
```
css is still not minified though. From my (limited) understanding `minimize: true` option of css-loader (which is already there) is supposed to do the minification, but for some weird reason it doesn't.

**Comment by ducksoupdev** [comment]
Implemented with [v1.3.0](https://github.com/ducksoupdev/vue-webpack-typescript/tree/v1.3.0)

**Comment by AccessViolator** [comment]
That was quick :)
Haven't tried it yet, but had a look at the changeset - I actually expected uiv to be replaced with bootstrap-vue with upgrade to bootstrap 4, as there's no any mention of bs 4 in uiv repo...

**Comment by ducksoupdev** [comment]
Good point :) It would be a useful library to include when bootstrapping new projects. I haven't used it before but am happy to update the template accordingly.

**Comment by ducksoupdev** [comment]
Implemented with [v1.3.1](https://github.com/ducksoupdev/vue-webpack-typescript/tree/v1.3.1)

**Comment by AccessViolator** [comment]
Good job, thanks.

Pity you ended up removing PurifyCSS, feels like a step back in functionality. I suppose it didn't play well with bs4 css?

One small issue though - 3 components come with their own css files, which fail to load. Changing selectors in webpack configs fixes the propblem:
```diff
-    test: /\.scss$/,
+    test: /\.(css|scss)$/,
```

**Comment by ducksoupdev** [comment]
Yes it is a shame I had to remove purifyCSS. It was troublesome getting it set-up and would only get in the way when a project increased in complexity. We ended up removing it from an internal project for that reason. 

I prefer to optimise as much as possible and have resorted to a post build script to remove unwanted CSS in some projects. 

Thanks for the heads up about the missing CSS, will fix it and release.
